app/javascript/geoblacklight/downloaders/downloader.js

```diff
@@ -0,0 +1,60 @@
+export default class GeoBlackLightDownloader {
+    constructor(el, options) {
+        this.el = document.querySelector(el);
```

Comment:
Why is this necessary? It looks like you're passing in an element, not a selector.

Reply:
This file can be deleted completely; downloads are handled via the `DownloadController` stimulus controller now.

Reply:
seems like we don't need app/javascript/geoblacklight/downloader.js, also?  I will rebase.

Reply:
yep, that's right. rebase should fix it